files/default/sql/ddl/2.6.0__initial_tables.sql

```diff
@@ -2270,3 +2270,65 @@ CREATE TABLE IF NOT EXISTS `git_repository_remotes` (
                                           KEY `git_repository_fk` (`repository`),
                                           CONSTRAINT `git_repository_fk` FOREIGN KEY (`repository`) REFERENCES `git_repositories` (`id`) ON DELETE CASCADE
 ) ENGINE=ndbcluster AUTO_INCREMENT=6164 DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+
+CREATE TABLE IF NOT EXISTS `expectation_suite` (
+    `id` INT(11) NOT NULL AUTO_INCREMENT,
+    `feature_group_id` INT(11) NOT NULL,
+    `name` VARCHAR(63) NOT NULL,
+    `meta` VARCHAR(1000) DEFAULT "{}",
+    `data_asset_type` VARCHAR(50),
+    `ge_cloud_id` VARCHAR(200),
+    `run_validation` BOOLEAN DEFAULT TRUE,
+    `validation_ingestion_policy` VARCHAR(25),
+    `created_at` timestamp DEFAULT CURRENT_TIMESTAMP,
+    PRIMARY KEY (`id`),
+    CONSTRAINT `feature_group_suite_fk` FOREIGN KEY (`feature_group_id`) REFERENCES `feature_group` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
```

Comment:
So, expectation suites are tightly coupled to FGs.
That's ok, we should make it clear in the docs, though. 
Because we had previously said they are reusable over FGs.

Reply:
The idea is to allow multiple expectation suites per feature gorups in the future, so you can have one suite for data validation, one for drift monitoring etc.

Reply:
The previous reusability was a joke. cause you couldn't change the parameters of an expectation when reusing it and reapplying to another feature group. Also it was tightly coupled to the feature name. Now you can get the GE native object in the client, change it however you want and reuse it that way. This is possible because we always return GE objects instead of our own.